src/frontend/src/views/GroupMembersPage/components/MembersTab/index.tsx

```diff
@@ -27,13 +29,16 @@ const MembersTab = ({
   groupName,
   groupId,
   members,
-}: Props): JSX.Element => {
+}: Props): JSX.Element | null => {
   const [tabValue, setTabValue] =
     useState<SecondaryTabType>(initialSecondaryTab);
   const [isInviteModalOpen, setIsInviteModalOpen] = useState<boolean>(false);
   const router = useRouter();
   const { data: invitations = [] } = useGroupInvitations(groupId);
+  const { data: userInfo } = useUserInfo();
+  const currentUser = find(members, (m) => m.id === userInfo?.id);
 
+  const isOwner = currentUser?.isGroupAdmin === true;
```

Comment:
Nothing inherently wrong with this implementation, but it strikes me as weird that `isGroupAdmin` isn't just part of the `useUserInfo` data. I think it would be more reasonable to add this to the `/me` endpoint than needing to look in the general members info for the group to get that info.

In the case where we have other things we might want to display for group_admins but not general group members, we'd have to get this member list and compare in those situations as well, which seems weird when it's really just a question about the user, not all the members of the group.

Reply:
yeah totally agree with you! I was just doing the easy solution due to time crunch, i'll update the /me endpoint if time permits.